countlyCommon/countlyCommon/Helpers/TimeHelper.cs

```diff
@@ -41,5 +41,18 @@ public static long UnixTimeNow()
         {
             return ToUnixTime(DateTime.Now.ToUniversalTime());
         }
+
+        /// <summary>
+        ///Convert a Unix timestamp to DateTime
+        /// </summary>
+        /// <param name="unixTimeStamp"></param>
+        /// <returns>DateTime</returns>
+        internal static DateTime UnixTimeStampToDateTime(long? unixTimeStamp)
+        {
+            // Unix timestamp is seconds past epoch
+            DateTime dtDateTime = new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
+            dtDateTime = dtDateTime.AddSeconds((double)unixTimeStamp).ToLocalTime();
```

Comment:
Bug. Converting to "double" from "long" leads to loss of data. Rework this

Reply:
> Bug. Converting to "double" from "long" leads to loss of data. Rework this
SDK already converting double to long, m casting this long to double back.
well! i am going o remove this function.

![image](https://user-images.githubusercontent.com/8924406/121033679-d458b480-c7c5-11eb-87e5-43ad09d047c7.png)